packages/design-picker/src/components/theme-preview/index.tsx

```diff
@@ -87,6 +92,12 @@ const ThemePreview: React.FC< ThemePreviewProps > = ( {
 		}
 	}, [ inlineCss, isLoaded ] );
 
+	useEffect( () => {
+		if ( ! isUrlWpcomApi( url ) ) {
```

Comment:
I think we can make it an initial state, right? Or will the URL be changed after the component mounts?

```js
const [ isLoaded, setIsLoaded ] = useState( ! isUrlWpcomApi( url ) );
```